test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -147,14 +147,17 @@ bool UcxContext::init()
     ucp_params_t ucp_params;
     ucp_params.field_mask   = UCP_PARAM_FIELD_FEATURES |
                               UCP_PARAM_FIELD_REQUEST_INIT |
-                              UCP_PARAM_FIELD_REQUEST_SIZE;
+                              UCP_PARAM_FIELD_REQUEST_SIZE |
+                              UCP_PARAM_FIELD_ESTIMATED_NUM_EPS;
```

Comment:
maybe better omit this flag if num_clients is not specified? (to rely on the default behavior even though it is currently the same)

Reply:
I could do it by setting `max_num_eps` to `UINT_MAX` and it will be considered as no `UCP_PARAM_FIELD_ESTIMATED_NUM_EPS` should be specified, wdyt?